modules/setting/setting.go

```diff
@@ -736,7 +736,7 @@ func NewContext() {
 
 	SSH.AuthorizedPrincipalsAllow = sec.Key("SSH_AUTHORIZED_PRINCIPALS_ALLOW").Strings(",")
 	if len(SSH.AuthorizedPrincipalsAllow) == 0 {
-		SSH.AuthorizedPrincipalsAllow = []string{"email", "username"}
+		SSH.AuthorizedPrincipalsAllow = []string{"username", "email"}
```

Comment:
what's the rationale for this change?

Reply:
usernames are quicker to check than emails